Scrapers/CompanyScrapers/GoarcScraper.py

```diff
@@ -0,0 +1,32 @@
+from Scrapers.Scraper import *
+
+TIMEOUT_IN_SECONDS = 30
+
+
+class GoarcScraper(Scraper):
+    url = 'https://go-arc.com/careers/'
+    name = 'Goarc'
+
+    def scrape(self):
```

Comment:
In general, your code is good but too "expensive" you're using two drivers and the scraper will run for longer than we'll want.
I've managed to do the same thing wihtout driver and the runtime is much much faster. take a look
```suggestion
    def scrape(self):
        soup = self.scraping_unit(self.url)

        for pos in soup.findAll('div', attrs={'class':'comeet-position-name'}):
            link = pos.findPrevious('a')['href']
            if "http" not in link:
                link = f"https:{link}"
            self.positions.append(
                self.Position(
                    title=pos.text.strip(),
                    link=link,
                    location=pos.findNext('div', attrs={'class':'comeet-position-meta'}).text.split('·')[0].strip(),
                )
            )
```